extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaHttpHandler.java

```diff
@@ -46,7 +46,7 @@
 
     private static final int BUFFER_SIZE = 8096;
 
-    private static Headers errorHeaders = new Headers();
+    private static final Headers errorHeaders = new Headers();
```

Comment:
I think code checkers will complain about the case as it's a constant (same for the other file).

Reply:
I don't get it, what would they complain about?

Reply:
Case for constants should be `LIKE_THIS`.

Reply:
Ah, the case. You mentioned it, but it just didn't register..